src/TextField/TextField.js

```diff
@@ -108,10 +123,14 @@ export default class TextField extends Component {
         )}
         <Input
           className={inputClassName}
-          value={value}
+          component={multiLine ? 'textarea' : 'input'}
```

Comment:
When I started making this change I realized it didn't make as much sense as I thought :(

The <Input /> component takes a `component` prop, and it feels wrong to sometimes determine the rendered component via the `component` prop and other times determine the rendered component via the `multiLine` prop. We'd have to do something like this: 

`const InputComponent = multiLine ? 'textarea' : ComponentProp;`

Which feels like unexpected behavior to me. Still happy to change this over if you like, but at this point I'm leaning towards keeping it the way it is.